
CLPY.EXCEPTION
===============================

.. cl:package:: CLPY.EXCEPTION

.. cl:function:: SET-CAUSE

.. cl:function:: RETURN-OR-RAISE-PYTHON-ERROR

.. cl:function:: GET

.. cl:function:: GET-CAUSE

.. cl:function:: FROM

.. cl:function:: RAISE-GENERIC-OR-PYTHON-ERROR

.. cl:function:: GET-TRACEBACK

.. cl:function:: SET-CONTEXT

.. cl:function:: NEW-EXCEPTION

.. cl:function:: TYPE

.. cl:function:: NAME

.. cl:function:: GET-CONTEXT

.. cl:function:: SET-TRACEBACK

